generator/generator_config.textproto

```diff
@@ -398,31 +398,31 @@ service {
   retryable_status_codes: ["kUnavailable"]
 }
 
-# GKE Hub
 service {
-  service_proto_path: "google/cloud/gkehub/v1/service.proto"
-  product_path: "google/cloud/gkehub"
+  service_proto_path: "google/cloud/gaming/v1/game_server_configs_service.proto"
```

Comment:
Looks like you also fixed some formatting / ordering problems in this file?

Reply:
The PR description seems to agree with you